### DISCUSSION TITLE
Making GPG Keyring usable

### DISCUSSION THREAD

**Discussion by aketchum15** [roddhjav/pass-import#148] [Q&A] [answered]
I was wondering if there could be a more in depth description of what it means to make the GPG keyring "usable" as stated in the docs. I currently have my ring set up with my private and public key and a few other public keys, but cannot get the program to run. I am having a hard time finding resources online as to what makes a keyring "usable." Thanks!

**Comment by roddhjav** [comment, marked as answer]
As explained in the Readme, usually it means setting the trust on your GPG key(s). So for all `gpgid` you have configured for pass in `.gpd-id`, you need to run: `gpg --edit-key <gpgid>`, then `trust` and set how much you trust the key. If you own the key, you can set you "trust it ultimately".
